app/main.py

```diff
@@ -1,3 +1,45 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from os import path
+
+from app.customer import Customer
+from app.shop import Shop
+from app.car import Car
+
+
+def shop_trip() -> None:
+    project_dir = path.dirname(path.dirname(path.abspath(__file__)))
+    config_file = path.join(project_dir, "app/config.json")
+    with open(config_file, "r") as file:
+        info_in_file = json.load(file)
+    itterable = 0
+    shop_list = []
+    for shop in info_in_file["shops"]:
+        shop_list.append(Shop(shop["name"],
```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one, the closing bracket is also written on a new line